app/models/orm/migrations/env.py

```diff
@@ -98,7 +98,7 @@ def run_migrations_online():
         with context.begin_transaction() as transaction:
             context.run_migrations()
             if "dry-run" in context.get_x_argument():
-                print("Dry-run succeeded; now rolling back transaction")
```

Comment:
Is this meant to be here?

Reply:
As I think you subsequently realized, this will disappear when I revert the dev-appeasing migration stuff.